funlet-whisper/tests/funlet-whisper.test.js

```diff
@@ -204,41 +192,41 @@ test('[WHISPER-OUTPUT-WHISPER-1-4] Human Check', () => {
 });
 
 test('[WHISPER-OUTPUT-WHISPER-2-0] No Digits Provided', () => {
-  const EMPTY_RESPONSE = XML_DECLARATION + '<Response/>';
-  let response = new Twilio.twiml.VoiceResponse();
-  let hasDigits = funlet.output.checkDigitsEntered(response, NO_DIGITS);
+  const EMPTY_RESPONSE = `${XML_DECLARATION}<Response/>`;
+  const response = new Twilio.twiml.VoiceResponse();
+  const hasDigits = funlet.output.checkDigitsEntered(response, NO_DIGITS);
   expect(hasDigits).toBe(false);
   expect(response.toString()).toEqual(EMPTY_RESPONSE);
 });
 
 test('[WHISPER-OUTPUT-WHISPER-2-1] A Digit was Pressed', () => {
-  const EMPTY_RESPONSE = XML_DECLARATION + '<Response/>';
-  let response = new Twilio.twiml.VoiceResponse();
-  let hasDigits = funlet.output.checkDigitsEntered(response, NON_EMPTY_DIGITS);
+  const EMPTY_RESPONSE = `${XML_DECLARATION}<Response/>`;
+  const response = new Twilio.twiml.VoiceResponse();
+  const hasDigits = funlet.output.checkDigitsEntered(
+    response,
+    NON_EMPTY_DIGITS
+  );
   expect(hasDigits).toBe(true);
   expect(response.toString()).toEqual(EMPTY_RESPONSE);
 });
 
 test('[WHISPER-OUTPUT-WHISPER-2-2] No Digits were Pressed, Empty Digits Set', () => {
-  const HANG_UP = XML_DECLARATION + '<Response>' + '<Hangup/>' + '</Response>';
-  let response = new Twilio.twiml.VoiceResponse();
-  let hasDigits = funlet.output.checkDigitsEntered(response, EMPTY_DIGITS);
+  const HANG_UP = `${XML_DECLARATION}<Response><Hangup/></Response>`;
+  const response = new Twilio.twiml.VoiceResponse();
+  const hasDigits = funlet.output.checkDigitsEntered(response, EMPTY_DIGITS);
   expect(hasDigits).toBe(true);
   expect(response.toString()).toEqual(HANG_UP);
 });
 
 test('[WHISPER-1-1] Full Response: Recorded Message', (done) => {
   const FULL_RESPONSE_WHISPER_1_1 =
-    XML_DECLARATION +
-    '<Response>' +
-    '<Gather numDigits="1">' +
-    '<Play>' +
-    RECORDED_MESSAGE +
-    '</Play>' +
-    '</Gather>' +
-    '</Response>';
-
-  const callback = (err, result) => {
+    `${XML_DECLARATION}<Response>` +
```

Comment:
all these multiline strings can be written to use a single starting \` and an ending `